zerver/lib/bugdown/__init__.py

```diff
@@ -856,6 +856,14 @@ def find_user_for_mention(self, name):
 
         return (False, user)
 
+    def check_outgoing_webhook_trigger(self, user):
+        TRIGGER_RE = re.compile(u'(@\*\*.*\*\* )(.*)')
+        if user["is_bot"]:
+            # Send the whole message as the command
+            command = TRIGGER_RE.search(current_message.content).group(2)
+            current_message.outgoing_webhook_triggers.append((user, command))
```

Comment:
This should probably check if it is an outgoing webhook bot, not just if it's a bot.